android/README.md

```diff
@@ -18,10 +18,19 @@ For MacOS/Linux users, add the following to **~/.bashrc** or **~/.bash_profile**
 export ANDROID_HOME=$HOME/Android/Sdk
 export PATH=$PATH:$ANDROID_HOME/tools
 ```
+For MacOS users, add the following (adjusted appropriately) to **~/.bashrc** or **~/.bash_profile** if your Java version is too strange for gradlew to understand (e.g., 11.0.2)
```

Comment:
The macOS changes look good.

Can you also update the Android Studio version in line 4 to `3.3.2+`?

Maybe specify elaborate line 5 to 
[Java SE Development Kit 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
That's what I'm using for Windows and Linux)